deploy/legionPipeline.groovy

```diff
@@ -90,9 +90,10 @@ def createjenkinsJobs(String commitID) {
     """
 }
 
-def runRobotTests() {
+def runRobotTests(String tags) {
```

Comment:
Maybe it will be better to add default value for compatibility with other installations where it is used?

Reply:
Done